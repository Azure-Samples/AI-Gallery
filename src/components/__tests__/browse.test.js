import React from 'react';
import renderer from 'react-test-renderer'
import Browse from './../browse';


test('renders a snapshot of browse', () => {
    const tree = renderer.create(<Browse/>).toJSON();
    expect(tree).toMatchSnapshot();
})