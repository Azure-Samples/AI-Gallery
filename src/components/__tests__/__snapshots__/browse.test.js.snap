// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot of browse 1`] = `
<div>
  <div
    class="content-container"
  >
    <div>
      <input
        class="button browsebtn noselect"
        id="OnnxSearch"
        onClick={[Function]}
        type="button"
        value="Onnx"
      />
    </div>
    <div>
      <input
        class="button  browsebtn noselect"
        id="TensorflowSearch"
        onClick={[Function]}
        type="button"
        value="Tensorflow"
      />
    </div>
  </div>
  <div
    id="browse_content"
  >
    <div
      class="browse_result"
      style={
        Object {
          "display": "inline-block",
          "verticalAlign": "top",
          "width": "100%",
        }
      }
    >
      <div
        className="gh-list"
      >
        <h3>
          Current Topic: '
          Onnx
          '
        </h3>
        <ul
          classname="gh-frame-container"
        />
      </div>
    </div>
  </div>
</div>
`;
